### ISSUE TITLE
Error installing Strongloop 6.0.1 with NVM: folder does not exist

### ISSUE THREAD

**Issue by frankorama** [strongloop/strongloop#321]
OS:  Ubuntu 14.04
Installed Node.js 6.2.2 using NVM.
I installed the required prerequisites for strongloop as per the web site's installation instructions.
When I attempt to install strlongloop v6.0.1
`npm install -g strlongloop@6.0.1`
I receive the following error:

```
npm ERR! Linux 3.13.0-95-generic
npm ERR! argv "/home/<username>/.nvm/versions/node/v6.2.2/bin/node" "/home/<username>/.nvm/versions/node/v6.2.2/bin/npm" "install" "-g" "strongloop@6.0.1"
npm ERR! node v6.2.2
npm ERR! npm  v3.9.5
npm ERR! path /home/<username>/.nvm/versions/node/v6.2.2/lib/node_modules/strongloop/node_modules/loopback-sdk-angular-cli/bin/lb-ng
npm ERR! code ENOENT
npm ERR! errno -2
npm ERR! syscall chmod

npm ERR! enoent ENOENT: no such file or directory, chmod '/home/<username>/.nvm/versions/node/v6.2.2/lib/node_modules/strongloop/node_modules/loopback-sdk-angular-cli/bin/lb-ng'
npm ERR! enoent ENOENT: no such file or directory, chmod '/home/<username>/.nvm/versions/node/v6.2.2/lib/node_modules/strongloop/node_modules/loopback-sdk-angular-cli/bin/lb-ng'
npm ERR! enoent This is most likely not a problem with npm itself
npm ERR! enoent and is related to npm not being able to find a file.
npm ERR! enoent 

npm ERR! Please include the following file with any support request:
npm ERR!     /home/<username>/npm-debug.log
npm ERR! code 1
```

**Comment by bajtos** [comment]
This issue should have been fixed in https://github.com/strongloop/strongloop/pull/318, which was released in `6.0.2`. Please `npm install -g strongloop@6.0.2`.

> I installed the required prerequisites for strongloop as per the web site's installation instructions.
> When I attempt to install strlongloop v6.0.1
> `npm install -g strlongloop@6.0.1`

I am wondering where is the version `6.0.1` coming from. Is this part of our documentation? (I hope not.) Did you @frankorama intentionally install a specific version of strongloop, instead of letting npm to pick the latest version?

cc @sam-github

**Comment by frankorama** [comment]
@bajtos yeah I did.  6.0.2 was causing problems with my code (backwards compatibility?) and I successfully installed 6.0.1 on my iMac & my Windows 10 laptop.  I wanted to continue coding my app on Linux, but suddenly couldn't install.  I'll install 6.0.2 and see what changes to my code I need to make.
